### PR TITLE
Feature/1624/cli enhancements get commands

### PR DESCRIPTION
**Pull request by checkelmann**
Resolves issue https://github.com/keptn/keptn/issues/1624

### CODE REVIEW

File: cli/cmd/get_project.go [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+)
+
+type getProjectStruct struct {
+	outputFormat *string
+}
+
+var getProject getProjectStruct
+
+var getProjectCmd = &cobra.Command{
+	Use:   "project",
```
Comment (bacherfl) [inline cli/cmd/get_project.go:42]: according to the Issue, the better approach would be to merge the command definitions for the singular and the plural into one command, using the `Aliases` property:

```
var getProjectCmd = &cobra.Command{
	Use:   "project",
        Aliases: []string{"projects"},
	Short: "Get details of a keptn project",
	Long:  `Get details of a given keptn project`,
	Example: `keptn get project sockshop
// ...
```

The same holds for the `get stage(s)` and `get service(s)` commands

**Comment by bacherfl** [comment]
Thank you for the contribution! I have added one remark in the comments about merging the singular and plural versions of the commands

File: cli/cmd/get_project.go (hunk 1 above)
Comment (checkelmann) [reply cli/cmd/get_project.go:42]: Ok, will look into it and try to merge these two commands into a single file, but not sure how to handle required arguments like --project and the different help text on the different sub-commands. 

If you have an example how I can do that would be good.

Comment (checkelmann) [reply cli/cmd/get_project.go]: Thank you @bacherfl, I've merged the commands.

**Comment by checkelmann** [comment]
Hi,

I'm done with my changes. This could be reviewed now.
@bacherfl @johannes-b

**Pull request merged**